src/Features/Core/Portable/EditAndContinue/AbstractEditAndContinueAnalyzer.cs

```diff
@@ -2805,6 +2805,13 @@ public ConstructorEdit(INamedTypeSymbol oldType)
                                             {
                                                 continue;
                                             }
+
+                                            // Deleting an ordinary method is allowed, and we store the newContainingSymbol in NewSymbol for later use
+                                            if (oldSymbol is IMethodSymbol { MethodKind: MethodKind.Ordinary, IsExtern: false, ContainingType.TypeKind: TypeKind.Class or TypeKind.Struct })
```

Comment:
>oldSymbol [](http://example.com/codeflow?start=48&length=9)

What about virtual/abstract methods?

Reply:
Not sure what you mean by this?

Deleting a virtual method could be a compile error if anything is overriding it, but even if that gets to the runtime, and the overridding method calls `base.Foo()`, it will throw MissingMethodException, which seems like it makes sense to me.

Reply:
Consider the following inheritance chain:
```C#
public class C { public virtual void M() => Console.WriteLine("C"); } 
public class D : C { public override void M() { base.M(); Console.WriteLine("D"); } } 
public class E : D { public override void M() { base.M(); Console.WriteLine("E"); } } 
```

If `D.M` is deleted I'd expect `E.M` to print `C` `E` and not throw.

Reply:
Perhaps we could emit `base.M()` to the body of a deleted method that overrides a base one, or report it as rude edit for now.

Reply:
Oh yes, of course, good point. Will make it a rude edit for now, and follow up.